site/content/docs/5.2/examples/features/index.html

```diff
@@ -292,13 +292,14 @@ <h3 class="fw-bold mb-0 fs-4">Featured title</h3>
     <h2 class="pb-2 border-bottom">Features with title</h2>
 
     <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-5 py-5">
-      <div class="d-flex flex-column align-items-start gap-2">
+      <div class="col d-flex flex-column align-items-start gap-2">
         <h3 class="fw-bold">Left-aligned title explaining these awesome features</h3>
         <p class="text-muted">Paragraph of text beneath the heading to explain the heading. We'll add onto it with another sentence and probably just keep going until we run out of words.</p>
         <a href="#" class="btn btn-primary btn-lg">Primary button</a>
       </div>
-      <div class="row row-cols-1 row-cols-sm-2 g-4">
-        <div class="d-flex flex-column gap-2">
+      <div class="col">
+        <div class="row row-cols-1 row-cols-sm-2 g-4">
+          <div class="col d-flex flex-column gap-2">
```

Comment:
There's an indentation issue here (inside this `<div>`); everything between l302 and l345 should have an extra-indentation.

Reply:
Hi @julien-deramond 

Formatting with EditorConfig has issues with my VS Code, tried Sublime Text as well.

It doesn't respect the EditorConfig, even when I don't have any other formatters/linters installed.

Is there any straight forward fix for this? 😊

Reply:
On my side I don't have anything to automatically format my code in VS Code but it's only a personal choice 😇 
In this case, just select all elements within this new `<div>` and add an indentation level in VS Code or any text editor; it should do it.

Reply:
Yeah, thanks, tried indenting it manually.

It should be fine now. Can you please check?

Reply:
> Just few modifications due to a previous automatic formatting I suppose. Could you please integrate these modifications? And it's going to be OK on my side for the review :)
> 
> ```diff
> diff --git a/site/content/docs/5.2/examples/features/index.html b/site/content/docs/5.2/examples/features/index.html
> index d32c446b3..35a4ef5da 100644
> --- a/site/content/docs/5.2/examples/features/index.html
> +++ b/site/content/docs/5.2/examples/features/index.html
> @@ -294,16 +294,14 @@ body_class: ""
>      <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-5 py-5">
>        <div class="col d-flex flex-column align-items-start gap-2">
>          <h3 class="fw-bold">Left-aligned title explaining these awesome features</h3>
> -        <p class="text-muted">Paragraph of text beneath the heading to explain the heading. We'll add onto it with another
> -          sentence and probably just keep going until we run out of words.</p>
> +        <p class="text-muted">Paragraph of text beneath the heading to explain the heading. We'll add onto it with another sentence and probably just keep going until we run out of words.</p>
>          <a href="#" class="btn btn-primary btn-lg">Primary button</a>
>        </div>
>  
>        <div class="col">
>          <div class="row row-cols-1 row-cols-sm-2 g-4">
>            <div class="col d-flex flex-column gap-2">
> -            <div
> -              class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
> +            <div class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
>                <svg class="bi" width="1em" height="1em">
>                  <use xlink:href="#collection" />
>                </svg>
> @@ -313,8 +311,7 @@ body_class: ""
>            </div>
>  
>            <div class="col d-flex flex-column gap-2">
> -            <div
> -              class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
> +            <div class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
>                <svg class="bi" width="1em" height="1em">
>                  <use xlink:href="#gear-fill" />
>                </svg>
> @@ -324,8 +321,7 @@ body_class: ""
>            </div>
>  
>            <div class="col d-flex flex-column gap-2">
> -            <div
> -              class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
> +            <div class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
>                <svg class="bi" width="1em" height="1em">
>                  <use xlink:href="#speedometer" />
>                </svg>
> @@ -335,8 +331,7 @@ body_class: ""
>            </div>
>  
>            <div class="col d-flex flex-column gap-2">
> -            <div
> -              class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
> +            <div class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
>                <svg class="bi" width="1em" height="1em">
>                  <use xlink:href="#table" />
>                </svg>
> ```

should be fine now.

Reply:
I'm so sorry to bother.